prow/manifests/overlays/metal3/config.yaml

```diff
@@ -960,17 +966,12 @@ presubmits:
     agent: jenkins
     always_run: false
     optional: true
-  - name: metal3-dev-env-integration-test-ubuntu-release-1-5
-    branches:
-    - release-0.4
-    agent: jenkins
-    always_run: false
-    optional: true
 
   metal3-io/cluster-api-provider-metal3:
   - name: gomod
     branches:
     - main
+    - release-1.8
```

Comment:
The definition for branch release-1.5 could be removed now for the affected jobs, right? That would reduce this massive config a little bit :smile:

Reply:
You're probably right, but I left them there since I wasn't sure :sweat_smile:

Reply:
Do adding 1.8 and removing 1.5 in two different PRs please. Much easier to review.